src/components/textfield/editForm/TextField.edit.data.js

```diff
@@ -46,4 +46,11 @@ export default [
       }
     ]
   },
+  {
+    weight: 205,
```

Comment:
This also needs to be added to the default schema or it will show up in the minified schema.